src/Pass/ImgTagTransformPass.php

```diff
@@ -110,6 +201,19 @@ protected function getImageWidthHeight($src)
 
         // Try obtaining image size without having to download the whole image
         $size = $this->fastimage->getImageSize($img_url);
```

Comment:
thumbor is open source.   It wouldn't be too strange to support it's filename format.